src/main/java/com/juniorstart/juniorstart/payload/LikerRequest.java

```diff
@@ -0,0 +1,18 @@
+package com.juniorstart.juniorstart.payload;
+
+import jakarta.validation.constraints.NotNull;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+public class LikerRequest {
```

Comment:
To ma odpowiadać za danie lajka tak? A skąd pobierane jest info kto wysyła tego lajka?